recipes-openxt/xctools/varstored-watch_git.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Provides binary that restarts varstored on failure"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=4641e94ec96f98fabc56ff9cc48be14b"
+
+DEPENDS = " \
+    xen-tools \
+"
+
+export INCDIR = "${STAGING_INCDIR}"
+export LIBDIR = "${STAGING_LIBDIR}"
+
+SRCREV = "${AUTOREV}"
+PV = "0+git${SRCPV}"
+
+S = "${WORKDIR}/git/varstored-watch"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```

Comment:
This is unnecessary - no local files in SRC_URI (and I think it's the default for a .bb)

Reply:
Good catch, this might have been a holdover.